src/js/backend.ts

```diff
@@ -5,47 +5,10 @@ import { Event, EventHint, Severity, Transport } from "@sentry/types";
 // @ts-ignore LogBox introduced in RN 0.63
 import { Alert, LogBox, YellowBox } from "react-native";
 
+import { ReactNativeOptions } from './reactNativeOptions';
```

Comment:
Could you please name the file `options.ts` to keep it consistent with the others?

```suggestion
import { ReactNativeOptions } from './options';
```

Reply:
done :)